client/src/templates/Introduction/intro.css

```diff
@@ -266,7 +266,7 @@ button.map-title {
 }
 
 .map-challenges-grid .map-challenge-title a:focus:not(:focus-visible) {
-  outline: none;
+  outline-color: transparent;
```

Comment:
```suggestion
```

Revert back to `outline: none`. This is for non-keyboard focus in which we don't want a focus outline.